documentation/docs/15-types.md

```diff
@@ -115,7 +115,14 @@ Others are required for SvelteKit to work properly, and should also be left unto
 		// TypeScript cannot 'see' when you
 		// use an imported value in your
 		// markup, so we need this
-		"preserveValueImports": true
+		"preserveValueImports": true,
+
+		// This ensures both SvelteKit
+		// and SvelteKit package build correctly
```

Comment:
"SvelteKit and SvelteKit" sounds a bit funny to me. I guess it would be the same as this:
```suggestion
		// This ensures both `svelte-kit build`
		// and `svelte-kit package` work correctly
```